components/13-dexidp/values.yaml

```diff
@@ -30,17 +30,57 @@ config:
         domain: default
         keystoneUsername: demo
         keystonePassword: DEMO_PASS
+    - type: oidc
+      name: azure
+      id: azure
+      config:
+        issuer: https://login.microsoftonline.com/570057f4-73ef-41c8-bcbb-08db2fc15c2b/v2.0
```

Comment:
Can we make the tenant UUID a variable like `$AZURE_SSO_TENANT_ID` like the other bits?

Reply:
sure, done in [fbab834](https://github.com/rackerlabs/understack/pull/23/commits/fbab834515c629402476b954c2cff31131855f89)